.github/workflows/ci.yml

```diff
@@ -30,6 +30,9 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements_dev.txt
+    - name: Sentry
+      run: |
+        echo "SENTRY_DSN=${{ secrets.SENTRY_DSN }}" >> $GITHUB_ENV
```

Comment:
I would drop this for now, as we don't use it anywhere else, and I think it might cause some confusion.